projects/Mallard/src/components/issue/issue-title.tsx

```diff
@@ -29,11 +28,13 @@ const GridRowSplit = ({
         Pick<ViewStyle, 'paddingTop' | 'paddingVertical' | 'paddingBottom'>
     >
 }) => {
-    const width = metrics.gridRowSplit()
+    const { gridRowSplit } = useLiveMetrics()
```

Comment:
gonna try to refactor this as a `<withbreakpoint>` too so as to not mix up concepts